### ISSUE TITLE
Update base license text

### ISSUE THREAD

**Issue by jonasraoni** [pkp/pkp-lib#7818]
**Describe the problem you would like to solve**
Our current license header requires some maintenance. Perhaps we can adopt another style to make it simpler to maintain.
There are many license variants out there, even big companies have mixed styles across their codebase. Follow some examples:

- Only the starting year (Adobe, Google, Microsoft, Salesforce, IBM): https://github.com/microsoft/vscode/blob/main/LICENSE.txt
- Starting + last updated year (Apple, Oracle): https://github.com/mysql/mysql-server/blob/8.0/README
- No dates, only the authors (Microsoft): https://github.com/dotnet/aspnetcore/blob/main/LICENSE.txt
- No header at all, just a `LICENSE` file (GitHub)

**Describe the solution you'd like**
In case we have lawyers available, it might be worth to check with them if adding a "last updated date" is enough to extend the copyright (the repositories at https://github.com/sfu didn't give me any clues). If not, then I'd opt-in for the approach that includes only the starting year.

**Who is asking for this feature?**
Developer

**Comment by asmecher** [comment]
@jonasraoni, this is more about the code header than the license, right? The code headers don't contain the license (that's typically in `docs/COPYING` or `LICENSE`), but they do make reference to it, and also state copyright and some self-documentation (`@class`, `@ingroup`, `@brief`, etc).

**Comment by jonasraoni** [comment]
Yes, specifically about the copyright part of the code header :)
